src/bundles/curves/curves_webgl.ts

```diff
@@ -96,14 +97,16 @@ function initShaderProgram(
  * @param num - num + 1 vertices to be drawn
  * @param drawMode - mode of drawing between points of the curve
  * @param space - visualization method used to render curve
+ * @param angle - angle of view
  */
 function drawCurve(
   gl: WebGLRenderingContext,
   buffers: BufferInfo,
   programInfo: ProgramInfo,
   num: number,
   drawMode: 'lines' | 'points',
-  space: '2D' | '3D'
+  space: '2D' | '3D',
```

Comment:
Maybe not not but Im thinking its probably worth to throw this into a [enum](https://www.typescriptlang.org/docs/handbook/enums.html) type.